src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
-export interface Figure {
 
+export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: string;
```

Comment:
It's better to create separate types for color values and shape values and use it here instead of string type